core-processor/src/precharge.rs

```diff
@@ -403,36 +444,15 @@ pub fn precharge_for_memory(
     }
 }
 
-#[cfg(test)]
-mod tests {
```

Comment:
having some tests would be great!

Reply:
Test for static pages was removed, because we removed charging for `static pages`

Several tests have already been added or changed by the PR. Feel free to check them.

Reply:
please remove this macros and use one function with inside:
```
match section_name {
    SecionName::Code => ...,
    ....
```

Reply:
Fixed